src/Mod/Path/App/Area.cpp

```diff
@@ -1325,6 +1298,9 @@ int Area::project(TopoDS_Shape &shape_out,
     area.myParams.Fill = TopExp_Explorer(shape_in,TopAbs_FACE).More()?FillFace:FillNone;
     area.myParams.Coplanar = CoplanarNone;
     area.myProjecting = true;
+    if (work_plane) {
```

Comment:
Avoid deriving the working plane from shapes by passing it down from the higher level operation. This avoids bombing out when some tiny surfaces or edges were generated by the projection, and strict coplanarity is not needed (per AreaParams.Coplanar).